asab/web/tenant/service.py

```diff
@@ -76,6 +76,10 @@ def _prepare_providers(self):
 			from .providers import WebTenantProvider
 			self.Providers.append(WebTenantProvider(self.App, self, Config["tenants"]))
 
+		if Config.get("zookeeper", "servers", fallback=None):
```

Comment:
i think you should rather test for `[tenants] zk_path` since you do not have a default value for that, to avoid a breaking change. atleast for now.

eventually we could probably introduce dedicated sections, such as `[tenants:zookeeper]`, `[tenants:web]` etc.